coalib/bears/Bear.py

```diff
@@ -99,7 +99,8 @@ class Bear(Printer, LogPrinter):
     and the CAN_DETECT sets. Possible values:
 
     >>> CAN_DETECT = {'Syntax', 'Formatting', 'Security', 'Complexity', 'Smell',
-    ... 'Redundancy', 'Simplification', 'Variable Misuse', 'Spelling', 'Other'}
+    ... 'Unused Code', 'Simplification', 'Variable Misuse', 'Spelling',
+    ... 'Memory Leak', 'Documentation', 'Duplication'}
```

Comment:
@sils1297 what is your opinion on this though? i think @justuswilhelm 's opinion is great, those are worth adding, but they are kind of more specific than what we already have. shall we choose more specific or leave abstract as now :) ?